### DISCUSSION TITLE
Who is using Sinatra?

### DISCUSSION THREAD

**Discussion by zzak** [sinatra/sinatra#1698] [General]
I sometimes ask myself who is still using Sinatra in production to a greater extent.

If you or your company are actively depending on Sinatra let us know!

Feel free to drop a link to your project or product and share how Sinatra works for you.

This may be useful for communicating upcoming changes in 3.x!

**Comment by dentarg** [comment]
We are proud users at [84codes](https://www.84codes.com/)! Sinatra is powering the web parts of [CloudAMQP](https://www.cloudamqp.com/), [CloudKarafka](https://www.cloudkarafka.com/), [ElephantSQL](https://www.elephantsql.com/) and [CloudMQTT](https://www.cloudmqtt.com/).

**Comment by walro** [comment]
We use it at @twingly too. Powering [APIs](https://data.twingly.net), internal tools and [publicly facing sites](https://app.twingly.com).

**Comment by cyclotron3k** [comment]
Ruby gems says the 2.1.0 version has been downloaded 4.7 million times (and 158m times overall), so it seems quite a lot of people are still using it!

**Comment by bnadlerjr** [comment]
We actively use Sinatra for several of our internal backend systems @flatiron-labs.

**Comment by DannyBen** [comment]
For me, one of the key features of Sinatra is that it can be used as an "embedded" server.

It is my framework of choice when:

1. Developing a CLI utility that has a server component.
2. Developing internal web applications, where it is desired to avoid the complexities that come with Rails.

**Comment by jkowens** [comment]
I was pleasantly surprised how many times Sinatra has been mentioned here!

https://news.ycombinator.com/item?id=34354129

**Comment by thedanbob** [comment]
I just finished a project using Sinatra today, a login-protected file server. I’ve also used it before for a video streaming frontend. I’m a Rails developer but for some sites Rails is overkill, so it’s great to have Sinatra in the toolbox.

**Comment by cf-ka** [comment]
I use Sinatra at my work to build small APIs, for some sort of middleware between Applications or to build smaller Web-Applications.
It's less difficult to use/setup, not so heavy like Rails or Django, and it's easier to maintain for a small team or only me
I get all my work done with Sinatra + SQLite and fits all my needs. <3
